src/MediaWiki/Api/PropertyListByApiRequest.php

```
@@ -18,10 +18,7 @@
  */
 class PropertyListByApiRequest {
 
-	/**
-	 * @var RequestOptions
-	 */
-	private $requestOptions = null;
+	private ?RequestOptions $requestOptions = null;
```

PHPMD (2.15.0)

[warning] Avoid unused private fields such as '$requestOptions'. (undefined)

(UnusedPrivateField)